fragment_internal_test.go

```diff
@@ -275,7 +275,7 @@ func TestFragment_SetValue(t *testing.T) {
 		}
 
 		// Non-existent value.
-		if value, exists, err := f.value(100, 11); err != nil {
+		if value, exists, err := f.value(101, 11); err != nil {
```

Comment:
this doesn't seem right -- if we set 100 up above, we should still have 100 here, no?

Reply:
previously, we were passing bitDepth of 11 to a field whose bitDepth was set to 10. Since the `exists`  bitmap was at the end, this made it look like it didn't exist. Now that exists is always at the beginning, we're trying to get a value from a column (101) which never had a bit set.

Reply:
ohh! i get it. okay, that makes more sense now. thanks.